### PR TITLE
Update event feed

### PR DESCRIPTION
**Pull request by ikoan**

### CODE REVIEW

File: app/views/page_editions/templates/grad/school/index.html.slim [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+          { term: { "organizations_slugs" => school_slug } },
+          { term: { "event_collections_slugs" => "graduate-program-information-sessions" } },
+          { range: { end_time_index: { gte: "#{Time.zone.now.to_i}" }}}
+        ]
+      }, limit: 3) 
+    non_info_session_events = EventResult.filter_by({
+        "and" => [
+          { term: { "_type" => "event_occurrence" } },
+          { term: { "organizations_slugs" => school_slug } },
+          { "not": { "term": { "event_collections_slugs": "graduate-program-information-sessions" } } },
```
Comment (jesilz) [inline app/views/page_editions/templates/grad/school/index.html.slim]: This is throwing a syntax error. It needs to be a hash rocket `=>`

File: app/views/page_editions/templates/grad/school/index.html.slim [hunk 2]
```diff
@@ -1,12 +1,29 @@
 ruby:
-  if @source.present?
-    event_category_id = @data.get(:event_category_id).item
-    events = @source.events.future.published.any_in(site_category_ids: event_category_id).asc(:start_date).limit(3)
-  else
-    event = []
-  end
+  school_slug = @source.slug
+  school_page = @source.present?
```
Comment (halloffame) [inline app/views/page_editions/templates/grad/school/index.html.slim]: I suggest using `is_school_page` everywhere `school_page` is used. That implies it is a boolean. You should also check to make sure the source is actually a school, if that is the intent. So you could do...

```ruby
is_school_page = @source.present? && @source.is_a?(School)
```

File: app/views/page_editions/templates/grad/_grad_events.html.slim [hunk 3]
```diff
@@ -1,6 +1,9 @@
 ruby:
   events ||= []
-
+  info_session_events ||= []
+  non_info_session_events ||= []
+  school_page
```
Comment (halloffame) [inline app/views/page_editions/templates/grad/_grad_events.html.slim]: Set a default value. You should probably do

```ruby
is_school_page ||= false
```

File: app/views/page_editions/templates/grad/_grad_events.html.slim [hunk 4]
```diff
@@ -30,19 +33,56 @@ ruby:
...
-          - if image_url = event.try(:image_medium_url).presence
-            img src=image_url
-          h3.title = link_to event.title, event, target:'_blank'
-          - if event.subtitle.presence
-            h4.subtitle = event.subtitle
-          p.date
-            i.fa.fa-fw.fa-calendar
-            = event.start_date.strftime('%B %-d')
+- if school_page
+  h2.text-24 = link_to 'Upcoming Information Sessions', events_path
```
Comment (halloffame) [inline app/views/page_editions/templates/grad/_grad_events.html.slim]: I'm not sure how helpful just linking to the events_path is. If there is a way to pass them to the search results for the given department, I think that would be best. Example: https://www.biola.edu/events/search?department=school-of-education. My guess is you should add `rel="nofollow"` to that link tag as well if you are appending query parameters to it.

File: app/views/page_editions/templates/grad/_grad_events.html.slim [hunk 5]
```diff
@@ -30,19 +33,56 @@ ruby:
...
+  h2.text-24 = link_to 'Other Upcoming Events', events_path
+  - unless non_info_session_events.empty?
+    .row
+      - non_info_session_events.each do |event|
+        .col-sm-6.col-md-4
+          .grad-event-item
+            - if image_url = event.try(:image_medium_url).presence
+              img src=image_url
+            h3.title = link_to event.title, event_path(event), target:'_blank'
+            - if event.subtitle.presence
```
Comment (halloffame) [inline app/views/page_editions/templates/grad/_grad_events.html.slim]: I know it was already like this before, but just FYI, the typical pattern we use here is to use `.present?` if you are just checking true or false, and use `.presence` if you are assigning the value to a variable that gets used inside the `if` statement.

**Pull request merged**